### PR TITLE
Pull up to drivex

### PR DESCRIPTION
**Pull request by grawk**
removing webdriver commands to selenium-drivex in order that other view modules can be written with the same functionality as nemo-view easily. This PR will work with `selenium-drivex >= v1.0.9`

### CODE REVIEW

File: lib/locreator.js [hunk 1]
```diff
@@ -34,22 +33,11 @@ module.exports.addGenericMethods = function generics(nemo) {
     return drivex.visible(_by(nemo, splitLocator(locatorString)));
   };
   nemo.view._wait = function (locatorString, timeout) {
-    function waitReturnElement() {
-      return drivex.waitForElement(_by(nemo, splitLocator(locatorString)), timeout || 5000);
-    }
-
-    var wep = new nemo.wd.WebElementPromise(nemo.driver, waitReturnElement());
-    return wep;
+    return drivex.waitForElementPromise(_by(nemo, splitLocator(locatorString)), timeout);
```
Comment (nikulkarni) [inline lib/locreator.js]: I know this PR doesn't touch this, however I feel that we need to add `msg` argument as part of these `_wait` methods. So optionally users can provide error message when elements timeout.

File: package.json [hunk 2]
```diff
@@ -12,13 +12,13 @@
     "url": "https://github.com/paypal/nemo-view.git"
   },
```
Comment (nikulkarni) [inline package.json]: should we bump `nemo-view` to `1.0.4`?

File: lib/locreator.js (hunk 1 above)
Comment (grawk) [inline lib/locreator.js]: Filed this issue to track: https://github.com/paypal/nemo-view/issues/19

Comment (nikulkarni) [inline lib/locreator.js]: :+1: I'll add this

**Comment by nikulkarni** [comment]
Travis CI build is in progress for long time?

**Comment by grawk** [comment]
Travis has been slow today :( Anyway no hurry we can merge this tomorrow..

**Comment by nikulkarni** [comment]
ok :)

**Pull request merged**